lnx-engine/search-index/src/index.rs

```diff
@@ -7,10 +7,7 @@ use hashbrown::HashMap;
 use crate::query::{DocumentId, Occur, QueryData, QuerySelector};
 use crate::reader::{QueryPayload, QueryResults};
 use crate::structures::{
-    DocumentHit,
-    DocumentOptions,
-    DocumentValueOptions,
-    IndexContext,
```

Comment:
Did you run rustfmt over this?

Reply:
Yes, but I didn't know I had to use the nightly build to get the configuration right. Just re-ran with nightly and seems to be correct now.

Reply:
Yes, but I didn't know I had to use the nightly build to get the configuration right. Just re-ran with nightly and seems to be correct now.